server/.go-version

```diff
@@ -1 +1 @@
-1.22.6
+1.23.7
```

Comment:
Asking here for convenience but not blocking: do we (or should we) have a policy for bumping dot releases? E.g. In a week, a new security issue is fixed in `1.23.8`. Should we bump and recompile our ESR?

Reply:
If the security team raises an issue, then yes.

Reply:
/cc @esarafianou for visibility